applications/SwimmingDEMApplication/custom_utilities/L2_error_projection_utility.h

```diff
@@ -78,10 +87,8 @@ double GetL2VectorProjection(ModelPart& r_model_part)
             error.push_back(error_x);
```

Comment:
Could you not pre-reserve the memory for this vector insteadd of doing all those push_backs?